### PR TITLE
Add first container to portfolio

### PR DESCRIPTION
**Pull request by francisco-carrola**
Breaking Change: 

Finally i get my first experience of frontend and with a lot of help I finally printed Olá mundo to everyone.

Thank you Maria and Ana. ✨

### CODE REVIEW

File: README copy.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (mariaines00) [inline README copy.md:1]: Needs work

**Review by mariaines00 — APPROVED** [review]
BEAUTIFUL 🥇

**Comment by mariaines00** [comment]
The most sophisticated piece of code I've ever seen omg

**Review by anafigm — APPROVED** [review]
very good work 😸

**Pull request merged**